server/src/server.ts

```diff
@@ -1,10 +1,11 @@
-import express from "express";
+import express, {Request, Response} from "express";
```

Comment:
I don't think we need to import Request/Response here, but if you need it for some testing it's alright

Reply:
let's remove!